### PR TITLE
add policy discovery support for karmor

### PR DESCRIPTION
**Pull request by Ankurk99**
Usage: karmor discover --policy {cilium/kubearmor}

### CODE REVIEW

File: src/server/grpcServer.go [hunk 1]
```diff
@@ -101,14 +101,15 @@ func (s *workerServer) GetWorkerStatus(ctx context.Context, in *wpb.WorkerReques
...
 	if in.GetPolicytype() == "network" {
 		log.Info().Msg("Convert network policy called")
 		networker.InitNetPolicyDiscoveryConfiguration()
-		networker.WriteNetworkPoliciesToFile("", "", []types.Service{})
+		return networker.ViewNetworkPolicies(), nil
 	} else if in.GetPolicytype() == "system" {
 		log.Info().Msg("Convert system policy called")
 		sysworker.InitSysPolicyDiscoveryConfiguration()
-		sysworker.WriteSystemPoliciesToFile(in.GetNamespace(), in.GetClustername(), in.GetLabels())
+		return sysworker.ViewSystemPolicies(), nil
```
Comment (seswarrajan) [inline src/server/grpcServer.go]: Please change it from ViewSystemPolicies to GetSystemPolicies

File: src/server/grpcServer.go [hunk 2]
```diff
@@ -101,14 +101,15 @@ func (s *workerServer) GetWorkerStatus(ctx context.Context, in *wpb.WorkerReques
 }
 
 func (s *workerServer) Convert(ctx context.Context, in *wpb.WorkerRequest) (*wpb.WorkerResponse, error) {
+
 	if in.GetPolicytype() == "network" {
 		log.Info().Msg("Convert network policy called")
 		networker.InitNetPolicyDiscoveryConfiguration()
-		networker.WriteNetworkPoliciesToFile("", "", []types.Service{})
+		return networker.ViewNetworkPolicies(), nil
```
Comment (seswarrajan) [inline src/server/grpcServer.go]: Please change it from ViewNetworkPolicies to GetNetworkPolicies

**Review by seswarrajan — CHANGES_REQUESTED** [review]
In protobuf add only worker related protobuf files. Kindly ignore adding other protobufs which were not modified.

File: src/server/grpcServer.go (hunk 2 above)
Comment (Ankurk99) [reply src/server/grpcServer.go]: There are already functions for GetNetworkPolicies and GetSystemPolicies which are called inside these functions.

Comment (Ankurk99) [reply src/server/grpcServer.go]: I will rename them to GetNetPolicy and GetSysPolicy, if that's fine too?

**Review by nyrahul — APPROVED** [review]
We need filters to be added to the policies, but i guess we decided to add that in a separate PR.

**Review by nyrahul — APPROVED** [review]

**Comment by nyrahul** [comment]
@seswarrajan, can you please review this and comment/approve? Thanks

**Review by seswarrajan — APPROVED** [review]

**Pull request merged**